MobileWallet/Common/DeepLinkManager.swift

```diff
@@ -43,7 +43,6 @@ import UIKit
 enum DeeplinkType {
    case send(deeplink: String?)
    case showQR
-   case viewTransaction(deeplink: String)
 }
 
 let Deeplinker = DeepLinkManager()
```

Comment:
`UpperCamelCase` for property name: _[https://swift.org/documentation/api-design-guidelines/#general-conventions](Names of types and protocols are `UpperCamelCase`. Everything else is `lowerCamelCase`.)_

Reply:
`let deepLinker = DeepLinkManager()`
or
`let deeplinker = DeeplinkManager()`

Reply:
(could apply to other deep-link-related var/class/function names).